### DISCUSSION TITLE
How to run a script while I'm in interactive mode.

### DISCUSSION THREAD

**Discussion by LintaoAmons** [getgauge/taiko#2418] [General]
# How I run a script when I'm in interactive mode

# Senario

- Step 1

    After I run 
	```bash
	~/Desktop ❯ taiko 
	> openBrowser()
	```
	and launched a browser.

- Step 2

	I did something directly by clicking and typing with the browser.

- Step 3

	I enter a page, where I already have a script. So can I do something like

	```
	> load("/path/to/script.js")
	```

	to perform the actions recorded inside that `script.js`?

**Comment by zabil** [comment]
Taiko extends the node REPL so you should be able to the use the `.load` command

https://nodejs.org/api/repl.html#commands-and-special-keys

**Reply by LintaoAmons** [reply]
Thanks a lot!!
